src/components/LoginCard.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div>
+    <v-toolbar class="pt-2 blue darken-4">
+      <v-toolbar-title class="white--text "><h4>Login</h4></v-toolbar-title>
+    </v-toolbar>
+    <v-card>
+      <v-card-text class="pt-4">
+        <div>
+          <v-form v-model="valid" ref="form">
+            <v-text-field
+              outline
+              prepend-icon="mdi-email "
+              label="Username"
+              v-model="email"
+              :rules="emailRules"
+              required
+            ></v-text-field>
+            <!--mdi-visibility not working fix it-->
```

Comment:
Why's this comment here?

Reply:
the icon was not visible changed `mdi-visibility` to `mdi-eye` now it works. comment removed.